src/ui/TemplateMessageItemBody/index.tsx

```diff
@@ -192,10 +190,14 @@ export function TemplateMessageItemBody({
             logger.error('TemplateMessageItemBody | no reservation key found in view_variables: ', reservationKey, templateData.view_variables);
             throw new Error();
           }
-          result.filledMessageTemplateItemsList = getFilledMessageTemplateItemsForCarouselTemplate(
-            simpleTemplateDataList,
-          );
-          result.carouselItem = carouselItem;
+          result.templateVersion = cachedTemplate.version;
+          result.filledMessageTemplateItemsList = [{
+            type: carouselItem.type as any,
```

Comment:
```suggestion
            type: carouselItem.type,
```
What happen if we remove `as any` here?